README.md

```diff
@@ -283,3 +283,12 @@ Object returned for getMeta and list methods, object has the following propertie
 	'modified': Wed Jan 01 2014 15:11:07 GMT+0100 (W. Europe Standard Time)
 }
 ```
+
+
+# License
+MIT
+
+# Contributing
```

Comment:
Idea of having contribute header is great :)
Should we have it in the beginning instead? (as it is now, no one will see if it they haven't read all documentation)